#! /bin/sh /usr/share/dpatch/dpatch-run
## micro_httpd.8--hyphen-fix.diff.dpatch by  <jari.aalto@cante.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Hyphen fix

@DPATCH@

diff --git a/micro_httpd.8 b/micro_httpd.8
index 255425a..8167af0 100644
--- a/micro_httpd.8
+++ b/micro_httpd.8
@@ -1,15 +1,15 @@
-.TH micro-httpd 8 "15 March 1999"
+.TH micro\-httpd 8 "15 March 1999"
 .SH NAME
-micro-httpd - really small HTTP server
+micro\-httpd \- really small HTTP server
 .SH SYNOPSIS
-.B micro-httpd
+.B micro\-httpd
 .I directory
 .SH DESCRIPTION
 .PP
-.I micro-httpd
+.I micro\-httpd
 is a very small HTTP server.
 It runs from inetd, which means its performance is poor.
-But for low-traffic sites, it's quite adequate.
+But for low\-traffic sites, it's quite adequate.
 It implements all the basic features of an HTTP server, including:
 .TP 3
 *
@@ -19,7 +19,7 @@ Security against ".." filename snooping.
 The common MIME types.
 .TP 3
 *
-Trailing-slash redirection.
+Trailing\-slash redirection.
 .TP 3
 *
 index.html
@@ -31,37 +31,37 @@ All in 150 lines of code.
 .PP
 To install it, add a line like this to /etc/inetd.conf:
 .nf
-    micro-http  stream tcp nowait nobody  /usr/local/sbin/micro-httpd micro-httpd dir
+    micro\-http  stream tcp nowait nobody  /usr/local/sbin/micro\-httpd micro\-httpd dir
 .fi
 Make sure the path to the executable is correct, and change "dir" to be
 the directory you want to serve.
 Then add a line like this to /etc/services:
 .nf
-    micro-http   port/tcp   #Micro HTTP server
+    micro\-http   port/tcp   #Micro HTTP server
 .fi
-Change "port" to the port number you want to use - 80, 8000, whatever.
+Change "port" to the port number you want to use \- 80, 8000, whatever.
 Then restart inetd by sending it a "HUP" signal, or rebooting.
 .PP
-On some systems, inetd has a maximum spawn rate - if you try to run
+On some systems, inetd has a maximum spawn rate \- if you try to run
 inetd services faster than a certain number of times per minute, it
 assumed there's either a bug of an attack going on and it shuts down
 for a few minutes.
-If you run into this problem - look for syslog messages about too-rapid
-looping - you'll need to find out how to increase the limit.
+If you run into this problem \- look for syslog messages about too\-rapid
+looping \- you'll need to find out how to increase the limit.
 Unfortunately this varies from OS to OS.
-On FreeBSD, you add a "-R 10000" flag to inetd's initial command line.
-On some Linux systems, you can set the limit on a per-service basis
+On FreeBSD, you add a "\-R 10000" flag to inetd's initial command line.
+On some Linux systems, you can set the limit on a per\-service basis
 in inetd.conf, by changing "nowait" to "nowait.10000".
 .PP
-Note that you can use micro-httpd to serve HTTPS, if you like, by running
+Note that you can use micro\-httpd to serve HTTPS, if you like, by running
 it from stunnel.
-First fetch and install stunnel - FreeBSD users can just go to
+First fetch and install stunnel \- FreeBSD users can just go to
 /usr/ports/security/stunnel and do a "make cert ; make install".
 Then as root run:
 .nf
-    stunnel -p /usr/local/certs/stunnel.pem -d 443 \
-	-l /usr/local/sbin/micro-httpd -- \
-	micro-httpd dir
+    stunnel \-p /usr/local/certs/stunnel.pem \-d 443 \
+	\-l /usr/local/sbin/micro\-httpd \-\- \
+	micro\-httpd dir
 .fi
 Make sure the paths to the certificate and executable are correct, and
 again don't forget to change "dir" to the directory you want to serve.
